aws/alarms/lambda/notify_slack/notify_slack.js

```diff
@@ -55,13 +58,61 @@ function getSNSMessageSeverity(message) {
   return "info";
 }
 
+function sendToOpsGenie(logGroup, logMessage, logSeverity, context) {
+
+  if (logSeverity !== 1 && logSeverity !== "SEV1") {
+    return; // skip sending to OpsGenie
+  }
+
+  var postData = {
+    message: logMessage,
+    entity: `*${logGroup}*`,
+    priority: "P1",
+  };
+
+  var options = {
+    method: "POST",
+    hostname: "api.opsgenie.com",
+    port: 443,
+    path: "/v2/alerts",
+    headers: {
+      "Content-Type": "application/json",
+      "Authorization": `GenieKey ${process.env.OPSGENIE_API_KEY}`,
+    },
+  };
+
+  var req = https.request(options, function (res) {
+    res.setEncoding("utf8");
+    res.on("data", function () {
+      console.log(
+        JSON.stringify({
+          msg: `Message successfully sent to OpsGenie... log level: ${logLevel}, log message: ${logMessage}`,
+        })
+      );
+    });
+  });
+
+  req.on("error", function (e) {
+    console.log(
+      JSON.stringify({
+        msg: `problem with request: ${e.message}`,
+      })
+    );
+  });
+
+  req.write(util.format("%j", postData));
+  req.end();
+
+}
+
 function sendToSlack(logGroup, logMessage, logLevel, context) {
   var environment = process.env.ENVIRONMENT || "Staging";
 
   const logLevelAsEmojiAndColor = (logLevel) => {
     switch (logLevel) {
       case "danger":
       case "error":
+      case "SEV1":
```

Comment:
Was this added as a preventive measure? At the moment we have no log level that is equal to `SEV1` in our code.